samples/Chat/src/app/utils/getExistingDisplayNameFromURL.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ *
+ * The displayName is extracted from the url
+ * using URLsearchparams.
+ *
+ * @returns The displayName as String
+ *
+ */
+
+export const getExistingDisplayNameFromURL = (): string | null => {
```

Comment:
I think each of these queryParams functions can live in the same file? `QueryParamLoginUtils.ts` or something